org-code-codebuilderapp/app/src/main/java/org/code/codebuilderapp/WebSocketConfig.java

```diff
@@ -0,0 +1,43 @@
+package org.code.codebuilderapp;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.messaging.simp.config.MessageBrokerRegistry;
+import org.springframework.web.socket.config.annotation.EnableWebSocketMessageBroker;
+import org.springframework.web.socket.config.annotation.StompEndpointRegistry;
+import org.springframework.web.socket.config.annotation.WebSocketMessageBrokerConfigurer;
+
+/**
+ * Configures websocket channels to send messages to specific users
+ */
+@Configuration
+@EnableWebSocketMessageBroker
+public class WebSocketConfig implements WebSocketMessageBrokerConfigurer {
+
+  /**
+   * Sets up the channels that a client can subscribe to. Specifically, a client
+   * would subscribe to /topic/destination and would send messages to
+   * /app/destination.
+   */
+  @Override
+  public void configureMessageBroker(MessageBrokerRegistry config) {
+    config.enableSimpleBroker("/topic");
+    config.setApplicationDestinationPrefixes("/app");
```

Comment:
Were the names "/topic" and "/app" picked because they're conventional?  (I'm wondering if we want to pick something that's a bit more descriptive.)

Reply:
Hah they're actually configuration settings (not sure why they're strings rather than enums...). 

From the [spring docs](https://docs.spring.io/spring-framework/docs/current/reference/html/web.html#websocket-stomp-message-flow): "For example, STOMP messages whose destination headers start with /app may be routed to @MessageMapping methods in annotated controllers, while /topic and /queue messages may be routed directly to the message broker."

Reply:
If we have control over these strings, it might be nice to pick something a bit more descriptive?

Reply:
✔️